theme/static/js/collection-resource.js

```diff
@@ -18,7 +18,8 @@ $(document).ready(function() {
     //                <th>Remove</th>          6
 
     var edit_mode = $("#edit-mode").val();
-    if (edit_mode.toLowerCase() == "false") {
+    var published = $("#published").val();
+    if (edit_mode.toLowerCase() === "false" || published.toLowerCase() === "true") {
```

Comment:
throughout the PR if `let` fulfills the need, it's preferable; we should be using `var` as sparingly as possible

Reply:
I assume I need to change only line#21. However, this js file has lot of variable declarations using  `var` which probably should have been `let`.  I am not planning to change those in this PR.

Reply:
Right, I would see those as pre-existing and appropriate for a separate pull request. This was fyi for future work and also noting, since other devs are using vuejs simple conditionals, such as v-if. Mauriel and I were/are not the only devs coding to this best practices approach. Obviously net new VueJS and larger efforts can be handled by me, but general participation at developer discretion is encouraged and the norm.